Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -132,4 +134,17 @@ public void setupDriveCommand(
         driveForDistance.includeOnSmartDashboard("Test drive for distance");
         gamepad.includeOnSmartDashboard("Change to GamePad Controls");
     }
+    
+    @Inject
+    public void setupAutonomous(
+            OperatorInterface oi,
+            DisableAutonomousCommand disable,
+            SetupDriveToHopperThenBoilerCommand driveToBoil
+            ){
+    disable.includeOnSmartDashboard();
+    driveToBoil.includeOnSmartDashboard();
+    
+    oi.leftButtons.getifAvailable(5).whenPressed(driveToBoil);
+    oi.rightButtons.getifAvailable(8).whenPressed(disable);
```

Comment:
I wouldn't even bind these to buttons for now. Just use the `includeOnSmartDashboard()` above.